configuration/external-storage.md

```diff
@@ -22,6 +22,12 @@ You can mount your HDD at a specific folder location. It is conventional to do t
     sudo apt-get update
     sudo apt-get install exfat-fuse
     ```
+5. If your HDD uses an NTFS file system, you will have read-only access to the HDD. If you want to write to the HDD, you can install the ntfs-3g driver:
```

Comment:
For the benefit of future editors, it might be nice to re-number the subsequent bullet points? :wink:

Reply:
I don't thint these are markup bullet numbers, otherwise you just use 1. for all of them and it renumbers automagically.